src/ensembl/src/content/home/Home.scss

```diff
@@ -1,44 +1,223 @@
 @import 'src/styles/common';
 
 .home {
-  padding-bottom: 80px;
+  display: grid;
+  grid-template-rows: minmax(550px, auto) 85px;
+}
 
-  h2 {
-    color: $grey;
-    font-size: 14px;
-    font-weight: $normal;
-  }
+.container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  background-color: $blue;
+  background-image: linear-gradient($black, $blue);
+  padding-top: 40px; 
+}
+
+.middleHomeWrapper {
```

Comment:
Mmm, I am not sure it's a middle of anything. `.mainTop` perhaps?

Reply:
Thanks for the suggestion. It's not easy to come up with class name which makes sense specially when you are in the middle of a code crisis :)